installer/scripts/omsadmin.sh

```diff
@@ -346,6 +362,28 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
By omsagent user they most likely meant the users that are used to perform operations within or by omsagent related process , so I would think processed spinned up by nxautomation will also count.